app/index.html

```diff
@@ -15,6 +15,14 @@
     <title>QIIME 2 Studio</title>
   </head>
   <body>
+    <div class="alert alert-danger text-center" style="margin-bottom:-35px">
+      DISCLAIMER: This is a functional prototype of a graphical user interface
+      for QIIME 2. This software is to be considered an alpha version, and may
+      undergo drastic changes at any time. Not all features of QIIME 2 are
+      available at this time via the Studio --- please see
+      <a href="https://docs.qiime2.org/2017.10/interfaces/q2cli/">q2cli</a>
```

Comment:
This maybe shouldn't be a link, it'll get outdated almost immediately, and if someone was able to install this, they probably already know about the CLI since all of our tutorials use it.